tests/simple/testdata/comparisons.textproto

```diff
@@ -250,6 +250,470 @@ section {
     value: { bool_value: false }
   }
 }
+section {
+  name: "Literal comparison for _<_"
+  description: "Comparing literals for the 'less than' operator"
```

Comment:
Add note that `a < b` should be equal to `b > a` and `!(a >= b)` and `!(b <= a)`.